docs/robots/boxer/integration_boxer.mdx

```diff
@@ -1,12 +1,81 @@
+import Support from "/components/support.mdx";
 
 <ComponentIntroductionBoxer />
 
-:::warning todo—under construction
+To attach custom hardware to Boxer, you will have to take care of mechanical mounting, electrical
+supply, and software integration. This guide aims to equip you with respect to these challenges.
 
-:::
+---
+
+### Mechanical Mounting
+
+When determining mechanical mounting, you can use the Standard mounting pattern on the top plate of the Boxer.
+At this time, the "PACS" mounting system is not available for Boxer.
+
+#### Mechanical, Standard
+
+External payloads can be attached to the mounting plate of the Boxer by using the predrilled Ø4.2 mm thru holes, using the Ø8 mm countersunk through holes on the underside, or adding new holes.
+Whichever method you choose, the method for removing the top plate is the same and outlined below.
+
+1. Ensure the robot is shut off, and the circuit breaker is turned to the _off_ position.
+2. Disconnect and remove all existing payloads on the top plate.
+3. Using a 2 mm hex key, remove the twelve M3 round head screws holding the connection covers in place.
+   Remove the panels.
+   Take care to disconnect the network cable and attachment cable from the robot when removing the connection covers.
+4. Using a 4 mm hex key, remove the eleven M5 socket head screws holding the plate down.
+5. Remove the plate from the Boxer.
+
+The Boxer plate is made from 8 mm aluminum and can be easily machined according to your customization needs.
+There are 19 premade Ø4.3 mm holes that can be used for mounting payloads to the Boxer.
+
+<center>
+  <img
+    src="/img/robot_images/boxer_images/robot_boxer_18.png"
+    width="600"
+  />
+</center>
+
+The plate also has eight Ø8 mm holes in the corners which are countersunk from the underside of the plate, making them ideal for flathead screws installed to protrude from the plate.
+
+<center>
+  <img
+    src="/img/robot_images/boxer_images/robot_boxer_19.png"
+    width="600"
+  />
+</center>
+
+### Electrical Integration
+
+For connecting with any external payloads, such as a backpack computer, refer to the product's datasheet or manual.
+If your robot was equipped with these payloads by Clearpath Robotics this documentation is provided along with your Boxer.
+Contact [Support](#support) if you need more information.
+
+Ensure that the Boxer is shut off and the circuit breaker is turned to the _off_ position before performing any electrical work on the Boxer.
+
+The Boxer is equipped with an attachment interface that consists of a Power-over-Ethernet (PoE) RJ45 bulkhead connector, a USB Type-A bulkhead connector, and a 37-position panel receptacle and mated plug.
+To connect with the Boxer attachment interface see Section 10 of the [OTTO 100 V2.4 Operation and Maintenance Manual](https://help.ottomotors.com/docs/files/53683025/99909709/1/1606232510624/OMM-000094-Operation+and+Maintenance+Manual+OTTO+100+V2.4_A.pdf).
```

Comment:
This link requires the User to login. Should we attach this content as a PDF to this Docs repository?

Reply:
I'm ok to require the login. The process for the Sales team when selling a Boxer is to ensure that each customer signs the OTTO EULA and gets an OTTO user account for help information, so Boxer owners should have access. I would prefer not to repost OTTO documentation in the public domain that is not already in the public domain. Thoughts?